### PR TITLE
Next upgrade

### PR DESCRIPTION
**Pull request by EdwinGuzman**
## Ticket:

No ticket, potentially to fix a file caching issue we're seeing in prod.

## This PR does the following:

- Upgrade next.js.

### CODE REVIEW

File: public/CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,19 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## Unreleased
+
+## [0.2.4] 2024-11-26
+
+### Updated
+
+- Updated Next.js to patch vulnerabilities.
```
Comment (7emansell) [inline public/CHANGELOG.md:14]: Include version number

**Review by 7emansell — APPROVED** [review]
Tests are passing locally so I think the Github fail is package issue

File: public/CHANGELOG.md [hunk 2]
```diff
@@ -44,16 +57,6 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - merged into 0.1.19
 
-## [0.1.19] 2024-11-18
```
Comment (avertrees) [inline public/CHANGELOG.md:47]: any particular reason why this was deleted

**Review by avertrees — APPROVED** [review]

File: public/CHANGELOG.md (hunk 2 above)
Comment (EdwinGuzman) [reply public/CHANGELOG.md:47]: not sure!

Comment (EdwinGuzman) [reply public/CHANGELOG.md:47]: this file was synced from the main changelog so this is now correct.

**Pull request merged**